govc/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # changelog
 
+### 0.25.0 (2021-04-16)
```

Comment:
the new version behavior (-l option) is missing

Reply:
I'll add that.  There's other stuff missing too.  Need a better way that going back through git history for future releases.
I didn't bother with the top-level govmomi/CHANGELOG.md this time and have never maintained one for vcsim.